src/ccs/lcccs.rs

```diff
@@ -145,8 +143,10 @@ mod tests {
     let (ccs, _, _, mles) = CCS::<G>::gen_test_ccs(&z);
     let ck = ccs.commitment_key();
 
+    let r_x: Vec<G::Scalar> = (0..ccs.s).map(|_| G::Scalar::random(&mut rng)).collect();
```

Comment:
Maybe this should be called `OsRng` for consistency? So that it's clear when elements come from a transcript and when from the OS?

Reply:
Normally in Rust, you follow the same way I did to use randomness. You call it just `rng` and move on. Indeed, the type is dependant on what you use. 
Anything that implements `RngCore` works. So it's not necessarily tight to the OS.